src/megachatapi.h

```diff
@@ -3109,6 +3108,19 @@ class MegaChatApi
      */
     MegaChatMessage *getMessage(MegaChatHandle chatid, MegaChatHandle msgid);
 
+    /**
+     * @brief Returns the MegaChatMessage specified from the chat room stored in node history
+     *
+     * This function allows to retrieve only those messages that are in the node history
+     *
+     * You take the ownership of the returned value.
+     *
+     * @param chatid MegaChatHandle that identifies the chat room
+     * @param msgid MegaChatHandle that identifies the message (msg id or a temporal id)
+     * @return The MegaChatMessage object, or NULL if not found.
+     */
+    MegaChatMessage *getMessageFromNodeHistroy(MegaChatHandle chatid, MegaChatHandle msgid);
```

Comment:
There's a typo requiring refactoring: `getMessageFromNodeHistroy()` --> `getMessageFromNodeHistory()`

Reply:
Ok, I also think that it's a clearer solution